README.md

```diff
@@ -450,9 +450,18 @@ make
 ```
 ### Ubuntu Bionic
 
+Install Requirements:
```

Comment:
I'd rather we tell people to use the vendored LLVM and then they can use the supported LLVM 7.0.1